ignite/handlers/checkpoint.py

```diff
@@ -353,6 +370,57 @@ def _setup_checkpoint(self) -> dict:
             checkpoint[k] = obj.state_dict()
         return checkpoint
 
+    @staticmethod
+    def setup_filename_pattern(
+        with_prefix: bool = True,
+        with_score: bool = True,
+        with_score_name: bool = True,
+        with_global_step_transform: bool = True,
+    ) -> str:
+        """Helper method to get the default filename pattern for a checkpoint.
+        Args:
+            with_prefix (bool): If True, the `filename_prefix` with an underscore (filename_prefix_) will be added to the filename pattern.
+                By default the `filename_prefix` will not be appended to the filename pattern
+            with_score (bool): If True, this indicates that the score function is not provided, then it will look for ``with_score_name``
```

Comment:
@Joel-hanson seems like I didn't read attentively the last time the doscrint. Now, the description IMO is a bit cumbersome. Argument `with_score`:
> If True, this indicates that the score function is not provided

it looks very misleading, IMO. More intuitive approach is if `with_score`, then template will have `{score}`. 
Am I missing something ?

Reply:
yes, @vfdev-5 its a bit confusing. I think I had mentioned this earlier https://github.com/pytorch/ignite/pull/1127#discussion_r446535788. Sorry for the confusion I can update this asap I should have communicated this properly.

Reply:
Please, try to make it simple and intuitive as much as possible.